plugins/GSdx/GSDialog.cpp

```diff
@@ -166,10 +166,17 @@ void GSDialog::SetTextAsInt(UINT id, int i)
 
 void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting>& settings, int32_t selectionValue, int32_t maxValue)
 {
+	if (settings.empty())
+		return;
+
 	HWND hWnd = GetDlgItem(m_hWnd, id);
 
 	SendMessage(hWnd, CB_RESETCONTENT, 0, 0);
 
+	const auto is_present = [=](const GSSetting& x) { return selectionValue == x.value; };
```

Comment:
Oh I didn't understand that `[=]` was enough to catch all variables by value. It is much nicer this way.